transcom/mymove
# Update SC Move create to self assign counseling office

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20785)

## Verification Steps for Reviewers
These are to be checked by a reviewer.


### How to test

1. Login as a Service Counselor
2. Search for a customer using a customer name 
3. Create new move for the customer
4. using swagger navigate to /moves/{locator}
5. Enter move code of newly created move / verify transportationOfficeId value
